src/main/java/com/nitnelave/CreeperHeal/block/BurntBlockManager.java

```diff
@@ -73,42 +64,42 @@ public static void forceReplaceBurnt(WorldConfig worldConfig)
     {
         World world = Bukkit.getServer().getWorld(worldConfig.getName());
 
-        Iterator<CreeperBurntBlock> iter = burntList.iterator();
```

Comment:
Java 8 change. You can keep some cosmetic changes (variable names and such), but let's keep it Java 6 for now.